backend/src/main/java/com/ody/meeting/controller/MeetingController.java

```diff
@@ -31,4 +33,13 @@ public ResponseEntity<MeetingSaveResponse> save(
                 .body(new MeetingSaveResponse(1L, "우테코 16조", LocalDate.parse("2024-07-15"), LocalTime.parse("14:00"),
                         "서울 송파구 올림픽로35다길 42", "37.515298", "127.103113", 1, List.of(new MateResponse("오디")), "초대코드"));
     }
+
+    @Override
+    @GetMapping("/invite-codes/{inviteCode}/validate")
+    public ResponseEntity<Void> validateInviteCode(
+            @RequestHeader(HttpHeaders.AUTHORIZATION) String fcmToken,
+            @PathVariable String inviteCode
+    ) {
+        return ResponseEntity.ok().build();
```

Comment:
컨벤션 정한거 개행 부탁드려요오 `체이닝 메서드 - 한 줄에 하나의 점`
```suggestion
        return ResponseEntity.ok()
            .build();
```